### ISSUE TITLE
Mermaid breaks when variables end in 'v'

### ISSUE THREAD

**Issue by FrenchBen** [knsv/mermaid#276]
## Steps to reproduce

Create a simple graph:
```
{% mermaid %}
graph TD;

  A(First Box)
  B(Second Box)
  A-->B
{% endmermaid %}
```
Now change any variable to end with 'v' and you'll get errors:
```
{% mermaid %}
graph TD;

  Av(First Box)
  B(Second Box)
  Av-->B
{% endmermaid %}
```

**Comment by knsv** [comment]
v ... That is probably to to the semantics of v to be the arrow down in the graph defintion:
```
graph v
```
 This equals:
```
graph TB
```

What version do you see this in? The reason I ask is because this is an old issue that has been fixed in newer versions of mermaid.

http://knsv.github.io/mermaid/live_editor/#/edit/Z3JhcGggVEQ7CgogIEF2KEZpcnN0IEJveCkKICBCKFNlY29uZCBCb3gpCiAgQXYtLT5C

**Comment by FrenchBen** [comment]
@knsv Thanks for the quick reply - This is indirectly in the gitbook mermaid plugin, but seems to have been updated with the latest code. 
https://github.com/JozoVilcek/gitbook-plugin-mermaid


I do like the live editor, it's working very nicely!

TL;DR:
I use this plugin: https://github.com/JozoVilcek/gitbook-plugin-mermaid
And mermaid isn't liking the 'v' - I have an issue on that repo too; since it seems to be fixed here, we can probably close this and blame the plugin :P

**Comment by knsv** [comment]
Thank, closing.
